src/main/java/teammates/common/datatransfer/SectionDisplayMode.java

```diff
@@ -0,0 +1,25 @@
+package teammates.common.datatransfer;
+
+/**
+ * Enum that defines supported displaying modes for section.
+ */
+public enum SectionDisplayMode {
+    GIVER_IN_SECTION("As long as giver is in section"),
+    RECIPIENT_IN_SECTION("As long as recipient is in section"),
+    BOTH_IN_SECTION("Both giver AND recipient are in section"),
+    GIVER_OR_RECIPIENT_IN_SECTION("Either giver OR recipient is in section");
```

Comment:
I think @damithc means that there should be the word `the` before each `giver` and `recipient`, e.g. the first one should be "As long as the giver is in section".